testapi.pm

```diff
@@ -115,16 +115,18 @@ Used for internal initialization, do not call from tests.
 =cut
 
 sub init {
-    if (get_var('SERIALDEV')) {
-        $serialdev = get_var('SERIALDEV');
+    if (check_var('BACKEND', 'ipmi')) {
+        $serialdev = 'ttyS1';
```

Comment:
The reason that drives me to make this change is that a aarch64 ipmi machine will use ttyAMA0 rather than ttyS1 according to convention. I am not sure whether ttyS1 also works for aarch64, maybe or maynotbe. So there is a risk I do not want to take. Better change the existing logic in this function. I do not object to your proposal of moving ipmi serialdev settings out of here totally. Do you need some other people to review or approve it further ?

Reply:
I'd accept the risk to rely on SERIALDEV purely (and only default to ttyS0 if not set)

Reply:
Double-checked on openqa.suse.de. It seems all IPMI machine got SERIALDEV set. I will make corresponding code change here if you nod.

Reply:
*nod*

Reply:
Change has already been made.